airflow/config_templates/default_airflow.cfg

```diff
@@ -98,7 +98,7 @@ sql_alchemy_pool_size = 5
 # additional connections will be returned up to this limit.
 # When those additional connections are returned to the pool, they are disconnected and discarded.
 # It follows then that the total number of simultaneous connections the pool will allow is pool_size + max_overflow,
-# and the total number of “sleeping” connections the pool will allow is pool_size.
+# and the total number of "sleeping" connections the pool will allow is pool_size.
```

Comment:
I'm not sure if this is the problem or not - but in my attempt to run it locally this caused an error:

```
root@da37259f18eb:/# airflow webserver
Traceback (most recent call last):
  File "/usr/local/bin/airflow", line 21, in <module>
    from airflow import configuration
  File "/usr/local/lib/python2.7/dist-packages/airflow/__init__.py", line 31, in <module>
    from airflow.utils.log.logging_mixin import LoggingMixin
  File "/usr/local/lib/python2.7/dist-packages/airflow/utils/__init__.py", line 24, in <module>
    from .decorators import apply_defaults as _apply_defaults
  File "/usr/local/lib/python2.7/dist-packages/airflow/utils/decorators.py", line 34, in <module>
    from airflow import settings
  File "/usr/local/lib/python2.7/dist-packages/airflow/settings.py", line 36, in <module>
    from airflow.configuration import conf, AIRFLOW_HOME, WEBSERVER_CONFIG  # NOQA F401
  File "/usr/local/lib/python2.7/dist-packages/airflow/configuration.py", line 575, in <module>
    conf.read(AIRFLOW_CONFIG)
  File "/usr/local/lib/python2.7/dist-packages/airflow/configuration.py", line 317, in read
    super(AirflowConfigParser, self).read(filenames, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/backports/configparser/__init__.py", line 705, in read
    self._read(fp, filename)
  File "/usr/local/lib/python2.7/dist-packages/backports/configparser/__init__.py", line 1035, in _read
    for lineno, line in enumerate(fp, start=1):
  File "/usr/lib/python2.7/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 2734: ordinal not in range(128)
```

Reply:
It wasn't this (though I will fix that in #5471 - if someone would like to review that I can cherry-pick it in)

It was `TERM` environment variable not being set, meaning :


```
  File "/usr/local/lib/python2.7/dist-packages/airflow/bin/cli.py", line 2186, in should_color
    if "color" in os.environ.get("TERM"):
TypeError: argument of type 'NoneType' is not iterable
```

Reply:
🥇